src/Commands/ListAddListCommand.ts

```diff
@@ -0,0 +1,31 @@
+import { NativeCallProperties } from "../Languages/Properties/NativeCallProperties";
+import { NativeCallCommand } from "./NativeCallCommand";
+import { Parameter } from "./Parameters/Parameter";
+import { SingleParameter } from "./Parameters/SingleParameter";
+
+/**
+ * A command for a list pop statement.
+ */
+export class ListAddListCommand extends NativeCallCommand {
+    /**
+     * Information on parameters this command takes in.
+     */
+    private static parameters: Parameter[] = [
+        new SingleParameter("name", "The name of the list.", true),
+        new SingleParameter("name", "The name of the list.", true)
+    ];
+
+    /**
+     * @returns Information on parameters this command takes in.
+     */
+    public getParameters(): Parameter[] {
+        return ListAddListCommand.parameters;
+    }
+
+    /**
+     * @returns Metadata on how to perform the native call. 
+     */
+    protected retrieveNativeCallProperties(): NativeCallProperties {
+        return this.language.properties.lists.concat;
+    }
+}
```

Comment:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file